AutonomousDemo/Loop.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AutonomousDemo
+{
+    public class Loop
+    {
+
+        public void GameLoop(bool answer)
+        {
+            int lastTime = DateTime.Now.Millisecond;
+            
+            while (true)
+            {
+                int start = DateTime.Now.Millisecond;
+                ProcessInput();
+                Update();
+                Render();
+                Thread.Sleep(900);
+            }
+
+        }
+
+        private void Render()
+        {
+            //show speedometer??
+            throw new NotImplementedException();
```

Comment:
You're probably not really going to 'render' anything but this might be a good time to write current velocity and position out to the console. (A better name might make more sense.)